bundles/org.openhab.binding.mycroft/src/main/java/org/openhab/binding/mycroft/internal/api/dto/MessageVolumeUnduck.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.mycroft.internal.api.dto;
+
+import org.openhab.binding.mycroft.internal.api.MessageType;
+
+/**
+ *
+ * @author Gwendal ROULLEAU - Initial contribution
+ */
+public class MessageVolumeUnduck extends BaseMessage {
+
+    public Data data = new Data();
+    public Context context = new Context();
+
+    public MessageVolumeUnduck() {
+        this.type = MessageType.mycroft_volume_unduck;
+    }
+
+    public static final class Data {
+    }
+
+    public static final class Context {
```

Comment:
What is the point of having this empty class?

Reply:
(Same)